site/de/spec/glossar.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: page
+lang: de
+lang-ref: glossary.md
+toc: true
+title: OCR-D Glossar
+---
+
+# OCR-D Glossar
+
+> Glossar von Begriffen aus dem Bereich der Bildverarbeitung/OCR und deren Verwendung im Rahmen von OCR-D
+
+Dieser Abschnitt ist nicht-normativ.
+
+## Layout und Typografie
+
+### Border
+
+Aus der [PAGE XML-Dokumentation](https://ocr-d.de/de/gt-guidelines/pagexml/pagecontent_xsd_Complex_Type_pc_BorderType.html)
+
+> Rand der eigentlichen Seite (wenn das gescannte Bild Teile enthält, die nicht zur Seite gehören)
+
+### Glyphe
+
+In OCR-D ist eine Glyphe die atomare Einheit innerhalb eines [Wortes](#wort).
+
+### Graphemhaufen / Grapheme Cluster
+
+Siehe [Glyphe](#glyph)
+
+### Lesereihenfolge (Reading Order)
+
+Die Lesereihenfolge beschreibt die logische Abfolge von [Regionen](#region) innerhalb eines Dokuments.
+
+### Line
+
+Siehe [TextLine](#textzeile)
+
+### PrintSpace
+
+Aus der [PAGE XML-Dokumentation](https://ocr-d.de/de/gt-guidelines/pagexml/pagecontent_xsd_Complex_Type_pc_PrintSpaceType.html)
+
+> Bestimmt die effektive Fläche einer gedruckten Seite auf dem Papier. Seine Größe ist für alle Seiten eines Buches gleich (Ausnahmen: Titelseite, mehrseitige Bilder).
+>
+> Sie enthält alle lebenden Elemente (außer Marginalien) wie Absätze und Überschriften sowie Fußnoten, Überschriften, laufende Titel.
```

Comment:
@tboenig Nur zur Sicherheit: Ist das korrekt? Ich dachte Marginalien wären in unserer Definition Teil des Satzspiegels.

Reply:
Ich würde sagen nein.

Reply:
@lena-hinrichsen preview words = catch word (see https://ocr-d.de/en/gt-guidelines/trans/lyKustode.html)
That's right, preview word is quite an unusual description. Better catch word. The function of this word or part of word is a preview (word) for the next page.

Reply:
> richtig, Marginalien gehören nicht dazu. siehe auch https://www.typolexikon.de/marginalie/

Yes, in the general definition that is correct, but don't we use border and print space interchangeably? E.g. eynollah produces a `pc:Border` that includes marginalia IIUC (@vahidrezanezhad?) but no PrintSpace. And in page-to-alto we fall back from PrintSpace to border  if the former is not present. In this regard, we diverge from the strict typographical definition and we should probably make this clear in our documentation.

Reply:
> Yes, in the general definition that is correct, but don't we use border and print space interchangeably?

Not in the glossary or other documentation, no. There used to be a time in the early days of phase 2 when that distinction had been overlooked during GT production, so out of sheer practical necessity in our implementation we had to treat the two interchangably. But this has been resolved long ago.

> E.g. eynollah produces a pc:Border that includes marginalia IIUC but no PrintSpace

Yes – so where is the contradiction to the above definition (i.e. our German translation of our interpretation of the PAGE documentation)?

> And in page-to-alto we fall back from PrintSpace to Border if the former is not present

The problem is that in ALTO, the two concepts are exclusive. Any region must be in either one of the `*Margin`s or in the `PrintSpace` – you simply **must have** a `PrintSpace` or you cannot represent any actual content. (Which is in stark contrast to PAGE's agnostic annotation where border+printspace+regions are all equal and voluntary peers.)

The sin in the current converter implementation is that we confound the meaning of the `*Margin`s by making them the space between the Border (i.e. leaf edges) and the image frame instead of the space between the Border and the PrintSpace. If we'd do the latter, then having no information on the PrintSpace would simply mean PrintSpace equals Border (i.e. empty Margins).

Reply:
Thanks for clearing this up @bertsky The german glossary is now part of spec.